### PR TITLE
Feat: navigation

### PR DESCRIPTION
**Pull request by Kavita539**
Added simple navigation

### CODE REVIEW

**Comment by Roshannair24** [comment]
Looks good to merge. Perhaps you would want to match the font size of the github icon with home and docs.

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+     font-family: 'Roboto Slab', serif;
+     font-weight: 500;
+     color: var(--primary-color);
+ }
+ 
+ 
+ .nav-logo-tagline {
+     font-size: 0.65rem;
+     font-weight: 500;
+     /* letter-spacing: 1px; */
```
Comment (SukanyaSen530) [inline components/navigation/navigation.css]: Suggestion: Can remove commented lines  if not needed.

**Review by SukanyaSen530 — COMMENTED** [review]
Look's good to merge :)

File: components/navigation/navigation.css (hunk 1 above)
Comment (Kavita539) [reply components/navigation/navigation.css]: ok will do

**Comment by Roshannair24** [comment]
looks good to merge.

**Review by Jia-A — COMMENTED** [review]
Everything looks good, maybe you can just use Beautify for formatting your code.

**Review by Roshannair24 — APPROVED** [review]

**Comment by Kavita539** [comment]
> Everything looks good, maybe you can just use Beautify for formatting your code.

yeah will do that later.

**Pull request merged**